2-extra/6-cheap-diner.js

```diff
@@ -19,7 +19,37 @@ chosenMeal(emptyArray)
 Should give the answer "Nothing :("
 **/
 
-function chooseMeal(mealArray) {}
+function chooseMeal(mealArray) {
```

Comment:
Question: Is there any simpler way this could be achieved by using a `sort` function?